apps/wakunode2/wakunode2.nim

```diff
@@ -97,15 +78,35 @@ when isMainModule:
   logging.setupLogLevel(conf.logLevel)
   logging.setupLogFormat(conf.logFormat, color)
 
-  info "Running nwaku node", version = app.git_version
-  logConfig(conf)
-
   case conf.cmd
   of generateRlnKeystore:
     doRlnKeystoreGenerator(conf)
   of noCommand:
+    # The Waku Network config (cluster-id=1)
+    if conf.clusterId == 1:
+      let twnClusterConf = ClusterConf.TheWakuNetworkConf()
+      if len(conf.shards) != 0:
+        conf.pubsubTopics = conf.shards.mapIt(twnClusterConf.pubsubTopics[it.uint16])
+      else:
+        conf.pubsubTopics = twnClusterConf.pubsubTopics
+
+      # Override configuration
+      conf.maxMessageSize = twnClusterConf.maxMessageSize
+      conf.clusterId = twnClusterConf.clusterId
+      conf.rlnRelay = twnClusterConf.rlnRelay
+      conf.rlnRelayEthContractAddress = twnClusterConf.rlnRelayEthContractAddress
+      conf.rlnRelayDynamic = twnClusterConf.rlnRelayDynamic
+      conf.rlnRelayBandwidthThreshold = twnClusterConf.rlnRelayBandwidthThreshold
+      conf.discv5Discovery = twnClusterConf.discv5Discovery
+      conf.discv5BootstrapNodes =
+        conf.discv5BootstrapNodes & twnClusterConf.discv5BootstrapNodes
+
     var wakunode2 = App.init(rng, conf)
 
+    info "Running nwaku node", version = app.git_version
+    logConfig(conf)
```

Comment:
In general, it looks great to me this PR but does it do any harm having the version shown above?

Reply:
it doesn't but i don't think there is any merit in printing that log while running the keystore generator